### DISCUSSION TITLE
How to enable user authentication with YUM install?

### DISCUSSION THREAD

**Discussion by kongbo96** [milvus-io/milvus#21373] [Q&A and General discussion]
How to enable user authentication with YUM install?
How does the modified configuration file take effect?

**Comment by Bennu-Li** [comment]
Milvus installed with YUM does not have user authentication enabled now. It is recommended to install Milvus using docker or k8s.

**Reply by xiaofan-luan** [reply]
@Bennu-Li Are we gonna to support it later?
